frontend/src/Components/Canada/BeforeArrival.js

```diff
@@ -1,13 +1,145 @@
 import React from 'react';
 import NavBar from "../Misc/NavBar";
 import Footer from "../HomePage/Footer";
+import '../../styles/BeforeArrival.css';
+import "../../../node_modules/bootstrap/dist/css/bootstrap.css";
+import "../../../node_modules/bootstrap/dist/css/bootstrap.min.css";
+//import Dropdown from 'react-bootstrap/Dropdown';
+
 
 class BeforeArrival extends React.Component {
   render() {
     return (
       <div>
         <NavBar/>
-        <div>Nothing for BeforeArrival yet</div>
+        <h1>Before Arrival</h1>
+        <div className="main-session">
+          <div className="container">
+            <h2><span>Step 1: Plan Your Quarantine</span></h2>
+            <br></br>
+            <div class="row align-items-center">
+              <div class="col">
+                <div class="info-card">
+                    <h3 class="h3">Assess your quarantine plan before you travel</h3>
+                    <a href="https://travel.gc.ca/travel-covid/travel-restrictions/isolation/quarantine-start">Learn more about the plan</a>
+                </div>
+              </div>
+              <div class="col">
+                  <div class="info-card">
+                    <p class="p1">Apply UofT Quarantine Plan</p>
+                    <ol>
+                      <li>Full Quarantine Accommodations</li>
+                      <li>Backup Quarantine Plan</li>
+                      <li>At-Home Quarantine</li>
+                    </ol>
+                    <a href="https://starportal.utoronto.ca/StarRezPortalX/3CF1ABA3/42/3919/Mandatory_Quarantine-Quarantine_Program_R" class="btn btn-primary">Online Application</a>
+                  </div>
+                </div>
+              </div>
+          </div>
+          
+          <br></br><br></br>
```

Comment:
it would be better to use margin-top / margin-bottom in css to adjust the whitespace in between elements. If you have to use `<br>`, replace `<br></br>` with `<br/>` for better coding style. Same applies to other places.